### PR TITLE
Add share speaker info function

### PR DESCRIPTION
**Pull request by leeweijie**
For GCI, by Lee Wei Jie

### CODE REVIEW

File: app/src/main/java/org/fossasia/openevent/activities/SpeakersActivity.java [hunk 1]
```diff
@@ -65,9 +68,22 @@ protected void onCreate(Bundle savedInstanceState) {
         sessionRecyclerView.setLayoutManager(new LinearLayoutManager(this));
         sessionRecyclerView.setAdapter(sessionsListAdapter);
         sessionRecyclerView.setItemAnimator(new DefaultItemAnimator());
+
+        shareButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                String speakerInfo = ("Speaker is: ".concat(speaker).concat("\n").concat("Speaker's email is: ").concat(selectedSpeaker.getEmail()));
```
Comment (creativepsyco) [inline app/src/main/java/org/fossasia/openevent/activities/SpeakersActivity.java]: Not Localization friendly plus has characters that can be hijacked to cause injection problems.

File: app/src/main/java/org/fossasia/openevent/activities/SpeakersActivity.java [hunk 2]
```diff
@@ -44,12 +47,12 @@ protected void onCreate(Bundle savedInstanceState) {
 
         selectedSpeaker = dbSingleton.getSpeakerbySpeakersname(speaker);
 
-
         TextView biography = (TextView) findViewById(R.id.speaker_bio);
         ImageView linkedin = (ImageView) findViewById(R.id.imageView_linkedin);
         ImageView twitter = (ImageView) findViewById(R.id.imageView_twitter);
         ImageView github = (ImageView) findViewById(R.id.imageView_github);
         ImageView fb = (ImageView) findViewById(R.id.imageView_fb);
+        FloatingActionButton shareButton=(FloatingActionButton) findViewById(R.id.share_speaker_button);
```
Comment (creativepsyco) [inline app/src/main/java/org/fossasia/openevent/activities/SpeakersActivity.java]: why the share is a floating action button. Mostly share is a top right button in the navigation bar.

File: app/src/main/java/org/fossasia/openevent/activities/SpeakersActivity.java (hunk 1 above)
Comment (leeweijie) [inline app/src/main/java/org/fossasia/openevent/activities/SpeakersActivity.java]: Ok wil fix it later

File: app/src/main/java/org/fossasia/openevent/activities/SpeakersActivity.java (hunk 2 above)
Comment (leeweijie) [inline app/src/main/java/org/fossasia/openevent/activities/SpeakersActivity.java]: Will change it

File: app/src/main/java/org/fossasia/openevent/activities/SpeakersActivity.java (hunk 1 above)
Comment (leeweijie) [inline app/src/main/java/org/fossasia/openevent/activities/SpeakersActivity.java]: Ok, new pull request here: https://github.com/fossasia/open-event-android/pull/168

**Pull request closed without merging**